LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -104,16 +80,12 @@ public IActionResult Index(Player player)
         /***
          * ListSpins Action
          **/
-
          public IActionResult LuckList(int id)
         {
-            //TODO: use the id to get the current player, including her Spins list
-
+            var Player = _dbc.Players.Include(x => x.Spins).Single(x => x.Id == id);
 
-            //TODO: Send the player's Spins to the View
-            return View();
+            return View(Player.Spins);
```

Comment:
This is not showing any spins for the player, because you added the spin to the Spin table (instead of the playerSpin) so the playerId was never set